app/views/decidim/devise/registrations/new.html.erb

```diff
@@ -45,7 +45,7 @@
             </div>
 
             <div class="field">
-              <%= f.password_field :password, help_text: t(".password_help", minimun_characters: NOBSPW.configuration.min_password_length), autocomplete: "off" %>
+              <%= f.password_field :password, help_text: t(".password_help", minimun_characters: ::PasswordValidator::MINIMUM_LENGTH), autocomplete: "off" %>
```

Comment:
Why did you update this line? It seems it creates a problem with the sign up form.
`uninitialized constant NOBSPW`

Reply:
It seems ok to me and to the CI